kuma/javascript/src/newsletter.jsx

```diff
@@ -36,15 +33,19 @@ export default function Newsletter() {
     const newsletterSubscribeURL = 'https://www.mozilla.org/en-US/newsletter/';
     const privacyPolicyURL = 'https://www.mozilla.org/privacy/';
```

Comment:
I don't know why these two are variables? Are they repeated? Can they don't just be part of the code directly? Or, if they're genuine constants, you might want to use the `const UPPER_CASE_NOTATION` on these to send a message to the code-reader that they never going to change.

Reply:
No, I guess they do not have to be. They are only used in one spot each.